requirements.txt

```diff
@@ -67,8 +67,8 @@ typing-extensions==4.4.0
 
 ## The following requirements were added by pip freeze:
 aiohttp==3.8.3
-aiosignal==1.2.0
```

Comment:
Did the sbom change need these requirements upgrades?

Reply:
No, but to trigger the "generate SBOM" action I needed to change requirements.txt, so I ran `make freeze-requirements`.

Now that we know it works, I could remove changes from requirements.txt from this PR if you would like.

Reply:
Actually, I'll just go ahead and do that. I don't want to mess up @andrewleith's PR.

Reply:
ah, I see. Yeah, I'd say lets remove the requirements changes to keep it just to fixing the SBOM action.